src/referrals-queue/referrals-queue.test.tsx

```diff
@@ -56,12 +56,11 @@ describe("referrals queue", () => {
     };
   }
 
-  let wrapper;
+  let wrapper: RenderResult;
```

Comment:
Since this is declared here and then set later, TS can't infer a type for it. Providing the type manually gives us VSCode code completion on `wrapper`'s methods.